src/js/components/ArtMenuItem.vue

```diff
@@ -17,46 +15,18 @@ export default {
     }
   },
   computed: {
-    ...mapGetters(['username', 'sessionID', 'userID'])
   },
   methods: {
-    fetchChildren() {
-
-      // we'll lazy load children, but cache them
-      if (this.lastFetch !== this.requestType[this.menuType]) {
-        this.$router.push({ name: 'workbenchScrollVersion',
-                            params: { scrollID: this.scrollID, 
-                                      scrollVersionID: this.versionID }
-        })
-
-        this.$post('resources/cgi-bin/GetImageData.pl', {
-        transaction: this.requestType[this.menuType],
-        combID: this.scrollDataID,
-        user: this.userID,
-        version_id: this.versionID,
-        SESSION_ID: this.sessionID
-      })
-      .then(res => {
-        if (res.status === 200 && res.data) {
-          this.children = res.data.results
-          this.lastFetch = this.requestType[this.menuType]
-        }
-      })
-      .catch(console.error)
-      }
-    },
-
     setRouter() {
-        this.$router.push({ name: 'workbenchScrollVersionTypeId',
+        this.$router.push({ name: 'workbenchAddress',
                           params: { scrollID: this.$route.params.scrollID,
                                     scrollVersionID: this.$route.params.scrollVersionID,
-                                    selectionType: 'art',
-                                    id: this.dataId, }
-      })
+                                    colID: this.$route.params.colID ? this.$route.params.colID : '-1',
+                                    imageID: this.$route.params.imageID ? this.$route.params.imageID : '-1',
+                                    artID: this.dataId }
+        })
```

Comment:
Github's code review format makes this a little difficult to see, but I suggest this formatting for these sort of nest ed objects:
```javascript
this.$router.push({ 
  name: 'workbenchAddress',
  params: {
    scrollID: this.$route.params.scrollID,
    scrollVersionID: this.$route.params.scrollVersionID
  },
  colID: this.$route.params.colID ? this.$route.params.colID : '-1',
  imageID: this.$route.params.imageID ? this.$route.params.imageID : '-1',
  artID: this.dataId
})
```